src/gps.cpp

```diff
@@ -7,7 +7,6 @@ namespace obc {
 namespace {
 
 constexpr auto baud_rate = 9600l;
-constexpr nmea_float_t velocity_conversion = 1.85166f;
```

Comment:
stała przeniesiona do hpp jednocześnie stała baudrate nadal jest w cpp?
ta nazwa mało mówi, konwersja miedzy jakimi jednostkami?
baudrate jakiego urządzenia? nie jest to w klasie gps ani namespace gps więc nazwa też powinna mówić coś więcej - gps_baud_rate?